test/plot-schema.json

```diff
@@ -16463,6 +16475,18 @@
       "inside bottom"
      ]
     },
+    "ticklabelrunoff": {
+     "description": "Shifts the tick labels by the specified number of pixels in parallel to the axis.",
```

Comment:
Also here let's clarify what the user should expect if a positive or negative value is used. 
Is it related to inside and outside tick position or the counter axis direction?

Reply:
Clarification was added.

Reply:
Thanks. Resolving...